source

```diff
@@ -10558,6 +10558,21 @@ partial interface <dfn id="document" data-lt="">Document</dfn> {
   <p>Each <code>Document</code> has a <dfn data-x="concept-document-navigation-id">navigation
   id</dfn>, which is a <span>navigation ID</span> or null, initially null.</p>
 
+  <p>Each <code>Document</code> has a <dfn
+  data-x="concept-document-bfcache-blocking-reasons"><a href="#note-bfcache">bfcache</a> blocking
```

Comment:
This might be ok, but I don't see examples of definitions that also link to something else like this. Maybe instead of directly linking bfcache here, add a note about what "bfcache blocking reasons" mean, and link bfcache here.